### DISCUSSION TITLE
"Select custom" > "Path" problems on Windows

### DISCUSSION THREAD

**Discussion by RayCulp** [qarmin/czkawka#639] [Q&A]
Sorry if this question has been asked/answered already, but I was unable to find any information in any of the issues on Github or in the documentation. 

I can't seem to get the "Select custom" > "Path" option working on Windows, no matter how I enter the path. I'm sure I must be doing something wrong. I have tried many combinations, such as:

```
c:\users\[username]\desktop\* (not case sensitive)

C:\users\[username]\Desktop\* (case sensitive)

c:\users\[username]\desktop* (no trailing backslash)

c:/users/[username]/desktop/* [Linux style forward slashes)

*desktop* (just to see if anything at all works)
```

None of these seem to work. Any advice would be greatly appreciated!

**Comment by chodthewacko** [comment]
It's just:
c:\users\[username]\desktop
you don't need the *    or the slash
